src/cephadm/cephadm

```diff
@@ -2217,12 +2219,16 @@ def get_config_and_keyring(ctx):
         keyring = d.get('keyring')
 
     if 'config' in ctx and ctx.config:
+        if not os.path.isfile(ctx.config):
+            raise Error('config file %s not found' % ctx.config)
```

Comment:
the next line calls `open()`, and that will already raise `FileNotFoundError`. No need to Look Before We Leap here

Reply:
ive put a try except around the open() call and raise an "Error"
![image](https://user-images.githubusercontent.com/22037319/112187855-d785d080-8bd8-11eb-86d4-4acc6b35be74.png)
same reasoning as above, i think its better to not show the full traceback, the message is enough for the user